src/dotnet/ProcessStartInfoExtensions.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Diagnostics;
+
+namespace Microsoft.DotNet.Cli
+{
+    internal static class ProcessStartInfoExtensions
+    {
+        public static int Execute(this ProcessStartInfo startInfo)
+        {
+            if (startInfo == null)
```

Comment:
Is it even possible for startInfo to be null in this case? Since this is an extension method?